src/scss/typography.scss

```diff
@@ -0,0 +1,57 @@
+////////////typography////////////////
+$spacer: 40px;
+
+$black: #000;
+$white: #fff;
+$aqua: #78d6e3;
+$green: #a8db57;
+$yellow: #d2cc66;
+$orange: #f6af65;
+$red: #e38a88;
+$gray: #4b4a4a;
+
+p {
+  font-family: "Montserrat", sans-serif;
```

Comment:
for body need to set default font, font color and font size

Reply:
@andch this should be defined in bootstrap variables: `$font-family-*` and `$font-weight-*` for keeping all bootstrap parts consistent. See https://github.com/twbs/bootstrap/blob/v4.6.0/scss/_reboot.scss